Dockerfile

```diff
@@ -6,6 +6,11 @@ ENV AIRFLOW_UID=501
 COPY requirements.txt ./requirements.txt
 COPY requirements-test.txt ./requirements-test.txt
 COPY dags ./dags
-
+RUN pip install wheel
+RUN python setup.py bdist_wheel 
+RUN pip install --no-cache-dir --upgrade setuptools && \
+    pip install --no-cache-dir --upgrade wheel && \
+    pip install --no-cache-dir --upgrade pip==22.0.4 && \
+    pip install --no-cache-dir --user -r requirements.txt -r requirements-test.txt
```

Comment:
you already have pip install wheel :) and why hardcode the pip version?

Reply:
I don't, it was one of the solutions I found. Looks like I overdid 😅